### PR TITLE
Add HAFS license file  LICENSE.md

### LINKED ISSUE #252: Need to add LICENSE.md
## Description
Provide a clear and concise description of the requested feature/capability.
Consider adding the LICENSE.md file.

## Proposed solution
How should the new feature/capability be added? If you have thoughts on the implementation strategy, please share them here.
See example from: https://github.com/ufs-community/ufs-weather-model/blob/develop/LICENSE.md

## Related to (optional)
Directly reference any related issues or PRs in this or other repositories, and describe how they are related. Examples:
- fixed by hafs-community/hafs/pull/<pr_number>

### PR DESCRIPTION
**Pull request by ZhanZhang-NOAA**
## Description of changes
This PR is to add a general license to HAFS repository. 

See for example: https://github.com/hafs-community/HAFS/blob/feature/hafs_license/LICENSE.md

## Issues addressed (optional)
- fixes hafs-community/HAFS/issues/252

## Tests conducted
N/A

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+This file is part of the Hurricane Forecast and Analysis System, referred to as HAFS, which is made available under version 3 of the Gnu Lesser General Public License.
+
+You may obtain a copy of the License at
+- HAFS: [GNU Lesser General Public License 3.0](https://www.gnu.org/licenses/lgpl-3.0.en.html) (LGPL)
+
+The following components used in HAFS are licensed separately: 
+- UFS: [UFS-weather-model License](https://github.com/ufs-community/ufs-weather-model/blob/develop/LICENSE.md)
```
Comment (gspetro-NOAA) [inline LICENSE.md]:
```suggestion
- UFS: [ufs-weather-model License](https://github.com/ufs-community/ufs-weather-model/blob/develop/LICENSE.md)
```

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- UFS-UTILS: [UFS-UTILS Licence](https://github.com/ufs-community/UFS_UTILS/blob/develop/LICENSE.md)
```
Comment (gspetro-NOAA) [inline LICENSE.md]:
```suggestion
- UFS_UTILS: [UFS_UTILS Licence](https://github.com/ufs-community/UFS_UTILS/blob/develop/LICENSE.md)
```

Comment (gspetro-NOAA) [reply LICENSE.md]: Alternatively, you could do:
```
- UFS Utilities: [UFS_UTILS Licence](https://github.com/ufs-community/UFS_UTILS/blob/develop/LICENSE.md)
```

File: LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+- GSI: [GSI Licence](https://github.com/NOAA-EMC/GSI-fix/blob/develop/LICENSE.md)
+
+The intent of this license is to ensure free and unrestricted access to the HAFS software, and to pass on those rights to modified versions this software.
```
Comment (gspetro-NOAA) [inline LICENSE.md]:
```suggestion
The intent of this license is to ensure free and unrestricted access to the HAFS software and to pass on those rights to modified versions this software.
```

File: LICENSE.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+Unless required by applicable law or agreed to in writing, software distributed under the License
```
Comment (gspetro-NOAA) [inline LICENSE.md]:
```suggestion
Unless required by applicable law or agreed to in writing, software distributed under this License 
```

**Comment by gspetro-NOAA** [comment]
Looks good to me! Just suggested a few tiny edits.

File: LICENSE.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+This file is part of the Hurricane Forecast and Analysis System, referred to as HAFS, which is made available under version 3 of the Gnu Lesser General Public License.
```
Comment (KathrynNewman) [inline LICENSE.md]:
```suggestion
This file is part of the Hurricane Analysis and Forecast System, referred to as HAFS, which is made available under version 3 of the GNU Lesser General Public License.
```

**Review by KathrynNewman — CHANGES_REQUESTED** [review]
Looks good to me as well! I added a couple additional minor edits.

**Review by BinLiu-NOAA — APPROVED** [review]
Looks good to me. Thanks!

**Comment by AvichalMehra-NOAA** [comment]
Approved.

**Review by KathrynNewman — APPROVED** [review]

**Pull request merged**